File: src/main/java/com/wei/Hello.java
package com.wei;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().hello();
        Person p = new Person(67f, 1.8f);
        p.hello();
       /* p.weight = 67f;
        p.height = 1.8f;*/
        System.out.println(p.bmi());

        int score = 88;
//        System.out.println(score >= 80);
//        System.out.println(score > 80 && score < 90);
        System.out.println(score < 80 || score > 90);

        char c = 'A';
        System.out.println(c > 'a');

        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "Wei";*/
    }
}

File: src/main/kotlin/com/wei/Hello.kt
package com.wei

fun main(args: Array<String>) {
    //println("Hello Kotlin")
    //Human().hello()
//    val h = Human(67f, 1.8f)
    val h = Human(weight = 67f, height = 1.8f)
    println(h.bmi())

    val score = 88
    println(score > 80)

    val c:Char = 'A'
    println(c.toInt() > 60)

    /*var age : Int = 19
    age = 20
    var weight = 66.5f
    var name : String
            name = "Wei"*/

}


class Human (var weight : Float , var height : Float , var name : String="") {

    init {
        println("test$weight")
    }

//    constructor(name:String , weight: Float , height: Float): this (weight, height)

    fun bmi():Float{
        val bmi = weight / (height*height)
        return bmi
    }

    fun hello(){
        println("Hello Kotlin")
    }

}